hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java

```diff
@@ -270,6 +271,11 @@ private OMConfigKeys() {
 
   public static final String OZONE_OM_METADATA_LAYOUT_PREFIX = "PREFIX";
 
+  public static final String OZONE_BUCKET_LAYOUT =
+      "ozone.bucket.layout";
```

Comment:
Can you please rename the config name to`ozone.default.bucket.type` or a better name conveying that the configurations meant for setting default value. Also, please add this config to the following files:

```
1) hadoop-hdds/common/src/main/resources/ozone-default.xml
2) hadoop-hdds/docs/content/feature/PrefixFSO.md
```

Reply:
Understood, will add this to the patch